src/connectedk8s/HISTORY.rst

```diff
@@ -3,6 +3,10 @@
 Release History
 ===============
 
+1.9.3
+++++++
+* Update for disconnected environment scenario.
```

Comment:
Include 2 more changes
- Update PyCryptoDome reference to 3.19.1
- Update ClusterDiagnosticChecks reference to the latest version - get ver from Nikko Miu